system.prop

```diff
@@ -3,3 +3,6 @@ ro.adb.secure=0
 #Disable for everyone for the moment
 ro.sys.sdcardfs=0
 persist.sys.overlay.nightmode=true
+persist.sys.usb.config=mtp,adb
+ro.debuggable=1
+ro.secure=0
```

Comment:
Like seriously, why?

Reply:
Hello
For MTP like default is persist.sys.usb.config=mtp,adb
without this line by default is ADB firstly
with this commit mtp firstly
2 next lines just good things, i'm use this commits on my OctopusOS